Apps/frontend/src/app.scss

```diff
@@ -38,6 +38,16 @@ img {
   height: 92%;
 }
 
+input[type="text"] {
```

Comment:
Since we have two inputs we could go for `input {  &[type="text"] { ... }  &[type="range] { ... }  }`

Reply:
This causes the sliders to revert to this:

![image](https://user-images.githubusercontent.com/81998094/213931163-ab6d97d6-b5a8-4953-9ed8-9d4a3fe12d49.png)

Reply:
And scaling is broken 
![image](https://user-images.githubusercontent.com/81998094/213931331-bd8189f9-8560-4aee-bfa5-b220a02987a3.png)


Ich würde es so belassen